libraries/render-utils/src/AntialiasingEffect.h

```diff
@@ -110,19 +110,20 @@ class AntialiasingConfig : public render::Job::Config {
     AntialiasingConfig() : render::Job::Config(true) {}
 
     enum Mode {
-        OFF = 0,
+        NONE = 0,
         TAA,
         FXAA,
         MODE_COUNT
     };
+    Q_ENUM(Mode) // Stored as signed int.
 
     void setAAMode(int mode);
     int getAAMode() const { return _mode; }
 
     void setDebugFXAA(bool debug) { debugFXAAX = (debug ? 0.0f : 1.0f); emit dirty();}
     bool debugFXAA() const { return (debugFXAAX == 0.0f ? true : false); }
 
-    int _mode{ TAA };
+    int _mode{ TAA }; // '_' prefix but not private?
```

Comment:
I'd move it. It doesn't seem to be referenced from anywhere, so this is likely not intentional.

Reply:
Ok, that's what I thought. I couldn't find any references to it but I'm not confident I would be able to confirm any problems in testing. I think it would be best to move it separately from this PR, but I can move it here if you want.